src/middlewared/middlewared/plugins/smb.py

```diff
@@ -421,6 +421,9 @@ async def getparm(self, parm, section):
                 return await self.getparm_file(parm)
         except MatchNotFound:
             pass
+        except FileNotFoundError:
+            self.logger.debug("%s: smb.conf file not generated. Returning default value.", parm)
```

Comment:
Just to be sure, would the return statement below be safe as the file couldn't be found ?

Reply:
Yeah, the loadparm context doesn't use this file.